src/components/input/input.js

```diff
@@ -170,7 +173,11 @@ function inputTextareaDirective($mdUtil, $window) {
     }
 
     var isErrorGetter = containerCtrl.isErrorGetter || function() {
-      return ngModelCtrl.$invalid && ngModelCtrl.$touched;
+      if (!ngModelCtrl.$touched) {
```

Comment:
Return true if input has not been touched, this causes the last value of $scope.watch to be true when the first digest is ran when the page loads. After the input has been touched then it will return the value of if the input is invalid.